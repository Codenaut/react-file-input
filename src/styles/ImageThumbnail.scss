.brainhub-file-input__thumbnail {
    grid-area: image;
    width: 40px;
}