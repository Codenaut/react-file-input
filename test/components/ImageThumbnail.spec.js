import React from 'react';
import renderer from 'react-test-renderer';

import ImageThumbnail from '../../src/components/ImageThumbnail';

describe('ImageThumbnail', () => {
  it('should match exact snapshot', () => {
    const tree = renderer.create(
      <div>
        <ImageThumbnail image="image_source"/>
      </div>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
