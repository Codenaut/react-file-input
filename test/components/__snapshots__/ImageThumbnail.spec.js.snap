// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageThumbnail should match exact snapshot 1`] = `
<div>
  <img
    className="brainhub-file-input__thumbnail"
    src="image_source"
  />
</div>
`;
